docker/Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN touch ${LOG_PATH}/django/api_app.log ${LOG_PATH}/django/api_app_errors.log \
     && rm -rf /root/.local
 
 # download yara rules and quark engine rules
-RUN api_app/analyzers_manager/repo_downloader.sh
+# RUN api_app/analyzers_manager/repo_downloader.sh
```

Comment:
this must be removed

Reply:
Yeah, I'm aware of this will revert it when PR is ready for review